### DISCUSSION TITLE
Unexpected margin on the video player

### DISCUSSION THREAD

**Discussion by zen-filip** [vidstack/player#1536] [Q&A]
Hey!

Just started using vidstack, and I noticed that there is an extra space underneath the video that showed up in my project but I also see it appearing on the official site. I would like to ask why is there extra space appearing and if there is a way to remove it?

<img width="955" alt="Screenshot 2024-12-15 at 17 22 00" src="https://github.com/user-attachments/assets/4964300e-4b5d-43ed-b9c5-0ba75b101c38" />

**Comment by mikevrpv** [comment]
Can confirm the same problem, in my case solved with setting up an outer div with overflow-hidden, proper aspect ratio and some other styling I needed in my layout. E.g. - class="w-full overflow-hidden aspect-video min-w-[275px] ... But there might be better ideas
